MarathonRecompLib/config/Marathon.toml

```
@@ -508,3 +508,33 @@ registers = ["r28", "f1"]
 name = "PlayerObject_ProcessMsgSuckPlayer_FixDeltaTime"
 address = 0x8219B660
 registers = ["r28", "f31"]
+
+[[midasm_hook]]
+name = "ParticleHFR_826454C8"
```

See https://github.com/sonicnext-dev/MarathonRecomp/pull/160/files#r2349206481.

```suggestion
name = "NOP"
```